### PR TITLE
Add documentation on support package feature

### PR DESCRIPTION
**Pull request by aHenryJard**
- Feature https://github.com/OpenCTI-Platform/opencti/issues/5548

Capture d'écran 2024-05-07 102356

### CODE REVIEW

File: docs/administration/support-package.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Support packages are usefull for troubleshooting issue that occurs on OpenCTI platform.
+Administrators can request to create and download a support package that contains recent platform error logs and usage statistics.
+
+Support Package can be requested from "Settings > Support" menu.
+
+![Support package overview](./assets/support-package-overview.png)
+
+1. On a click on "Generate support package" button on the top right, the support package is requested.
+1. A background task on the platform gather logs and interesting data and create an archive
```
Comment (yassine-ouaamou) [inline docs/administration/support-package.md:11]: A background task on the platform **gathers** logs and interesting data and **create** an archive

File: docs/administration/support-package.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Support Package
+
+Support packages are usefull for troubleshooting issue that occurs on OpenCTI platform.
+Administrators can request to create and download a support package that contains recent platform error logs and usage statistics.
+
+Support Package can be requested from "Settings > Support" menu.
+
+![Support package overview](./assets/support-package-overview.png)
+
+1. On a click on "Generate support package" button on the top right, the support package is requested.
```
Comment (yassine-ouaamou) [inline docs/administration/support-package.md:10]: All the elements of the list have 1. as a number

Comment (aHenryJard) [reply docs/administration/support-package.md:10]: yes it's intended, it's a trick on markdown to not have to change number if you insert a new item in the middle

File: docs/administration/support-package.md (hunk 1 above)
Comment (aHenryJard) [reply docs/administration/support-package.md:11]:
```suggestion
1. A background task on the platform gathers logs and interesting data and creates an archive

```

**Pull request merged**